src/app/tests/suites/certification/Test_TC_APPLAUNCHER_3_7.yaml

```diff
@@ -39,14 +43,13 @@ tests:
                 value: nodeId
 
     - label:
-          "Step 1: TH sends a LaunchApp command to DUT to launch and bring an
+          "Step 1a: TH sends a LaunchApp command to DUT to launch and bring an
           application to the foreground"
-      PICS: APPLAUNCHER.S.C00.Rsp
       command: "LaunchApp"
       arguments:
           values:
               - name: "Data"
-                value: "Hello World"
+                value: "DataValue"
```

Comment:
should this be DataValue and not "DataValue" so it references the value of DataValue declared on line 31?

Reply:
I still see quotes around "DataValue". Am I misunderstanding something?